### PR TITLE
feat[catalogue3]: reintroduce databanks and user interface fixes for vac4eu

### PR DESCRIPTION
**Pull request by mswertz**
To test:
1. create a schema vac4eu with as template catalogue3
2. upload the excels from https://drive.google.com/drive/folders/1XEO2kTiC2vlegJsiBNbUMw0CBfGHKi7t

New issues:
- [x] In Variables view, there is a space missing between Dataset name and the name of the dataset:
- [x] In Variables view show and hide NA does not work
- [x] Menu redirects to empty page, there seems to be a '/' missing in the url:
- [x] At Data sources cannot click I expect to be able to click on Linked datasources and be redirected to that data source.
- [x] When clicking on a link inside a resource that redirects to another resource, I expect to be redirected to the top of the page of that resource, e.g. at Organisation ARS.TOSCANA click on a 'Leading resource'.
- [x] Back to top redirects to empty page, there seems to be a '_' instead of a '/' in the url
- [x] In Data sources view the Linked data sources table is outside of the blue 'frame', see e.g.: https://preview-emx2-pr-1962.dev.molgenis.org/vac4eu/catalogue3/#/datasources/ARS

Checklist:
- [x] fixes data type for vac4eu and conception
- [x] change user interface to show databank type in refBack for linked data sources 
- [x] check that ETLs can now still be properly edited
- [x] fixing all screens for the changes (mainly pid->id and changes dictionary/table/source*/target* to dataset/variables)
- [x] mappings from and to should both be visible in dataset screen

Navigation issues:
- [x] At the Models view, I expect to be able to click on a linked Dataset and move to the Dataset view.
- [x] At Organisations I expect to be able to click on a resource and go to their view.
- [x] At Organisations I expect to be able to click on a link to the website.
- [x] At Organisations Resources.contacts are represented as a link which does not lead to anything, but does appear in the bread crumbs. **<= cannot reproduce this issues?**
- [x] At Networks, I can click on a Data source, but am not redirected to that data source, because the address is wrong (.... organisations/datasource_id)
- [x] At Datasets I expect to be able to click on a Dataset and see the Variables in that Dataset.

Data is slow to load:
- [x] When clicking catalogue, screen first shows all available tables in blocks and then shifts to show tables that are populated. Maybe a performance issue?
- [x] Same happens when opening a table, e.g. Cohorts: first you get a screen showing ‘no results’, that is followed by a change of the screen to show the table entries. 

Review findings (might not be urgent):
- [x] breadcrumbs should be all clickable
- [x] layout of bottom boxes different than boxes above

### CODE REVIEW

**Review by BrendaHijmans — APPROVED** [review]
datamodel changes are ok

**Review by BrendaHijmans — CHANGES_REQUESTED** [review]
UI views voor Datasources, Orgabisations etc are broken

**Comment by mswertz** [comment]
> UI views voor Datasources, Orgabisations etc are broken

Might this have to do with catalogue3 => I have fixed example data to use catalogue3

**Comment by BrendaHijmans** [comment]
At the models view, I cannot click on Dataset and then go to the Dataset

File: apps/catalogue3/src/components/detailView/EmailFieldValue.vue [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<template>
+  <a :href="'mailto:' + data">{{ data }}</a>
```
Comment (connoratrug) [inline apps/catalogue3/src/components/detailView/EmailFieldValue.vue:2]: why not leave it up to the data input , by setting the correct url the DM can choose to add a link or mailto or something else

File: apps/catalogue3/src/components/detailView/EmailFieldValue.vue [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+</template>
+
+<script>
+export default {
+  name: "EmailFieldValue",
+  props: {
+    data: {
+      type: [String, Object],
```
Comment (connoratrug) [inline apps/catalogue3/src/components/detailView/EmailFieldValue.vue:10]: Object ?

File: apps/catalogue3/src/components/detailView/EmailFieldValue.vue (hunk 1 above)
Comment (mswertz) [reply apps/catalogue3/src/components/detailView/EmailFieldValue.vue:2]: we have a field of type=email and therefore we have this

File: apps/catalogue3/src/components/detailView/EmailFieldValue.vue (hunk 2 above)
Comment (mswertz) [reply apps/catalogue3/src/components/detailView/EmailFieldValue.vue:10]: Indeed, that is wrong

**Comment by BrendaHijmans** [comment]
Nice to see that the progress of manual uploads is specified again.

**Comment by BrendaHijmans** [comment]
Navigation issues:
-Menu redirects to empty page:
image

**Comment by BrendaHijmans** [comment]
-Editing of variables in ETL view works.

**Comment by BrendaHijmans** [comment]
In Data sources view the Linked data sources table is outside of the blue 'frame', see e.g.: https://preview-emx2-pr-1962.dev.molgenis.org/vac4eu/catalogue3/#/datasources/ARS

**Review by BrendaHijmans — APPROVED** [review]

**Pull request merged**